packages/contentstack-utilities/package.json

```diff
@@ -27,7 +27,7 @@
   "license": "MIT",
   "dependencies": {
     "@contentstack/management": "^1.3.0",
-    "@oclif/core": "^1.8.2",
+    "@oclif/core": "^1.13.10",
```

Comment:
we can install the oclif and include the core package from oclif, why we need to install separate package if we already have oclif package installed at root.

Reply:
As oclif dev dependency package is different and does not include @oclif/core package we have install @oclif/core package.